devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -161,7 +161,9 @@ public void execute(Task test) {
 
                     Task classesTask = tasks.getByName(JavaPlugin.CLASSES_TASK_NAME);
                     Task resourcesTask = tasks.getByName(JavaPlugin.PROCESS_RESOURCES_TASK_NAME);
-                    quarkusDev.dependsOn(classesTask, resourcesTask, quarkusGenerateCode);
+                    Task testClassesTask = tasks.getByName(JavaPlugin.TEST_CLASSES_TASK_NAME);
```

Comment:
Note that this now means that quarkusDev will fail in gradle if the tests don't compile. In maven I can easily just ignore the failures, but I am not sure how to do that in Gradle.

Reply:
fyi @glefloch